docs/registry.md

```diff
@@ -0,0 +1,60 @@
+ # Registry Support for SOCI
+
+SOCI is supported by most registries. The notable exception to this is Duckerhub, which currently does not meet our technical requirements. 
+To check if your registry of choice is supported, see [#list-of-registry-support](List of Registry Support).
+
+For most use-cases, support is the only concern. However, there is a difference in *how* registries support SOCI that could cause surprising edge cases. The rest of this document is a technical dive into how SOCI indices are stored and retrieved from registries and the surprises you might encounter.
+
+## Registry Requirements
+
+In order for a registry to support SOCI it must have the following features:
+
+1) Accept [OCI Image Manifests](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md) with [subject fields](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md#image-manifest-property-descriptions) and arbitrary config media types.
+This allows the registry to store SOCI indices and is supported by most registries.
+
+2) (optional) Support the [OCI referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers)
+This adds convenience around retrieving SOCI indices from the registry. If it is not supported, there is a [fallback mechanism](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api) that can be used instead, but it has a few issues noted in the next section. Since the OCI referrers API was defined in the not-yet-released [OCI Distribution Spec v1.1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md), very few registries support it.
+
+## Referrers API vs Fallback
+
+Registries can support retrieving SOCI artifacts either though the [OCI referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers) or a [Fallback mechanism](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api). The referrers API is part of the not-yet-released [OCI Distribution Spec v1.1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md) so support is limited. The Fallback API is supported by all registries, but has notable edge cases.
```

Comment:
nit: SOCI artifacts -> SOCI artifacts (index, ztoc).

though -> through

Reply:
The 2nd sentence is repeated with the sentence above "2) (optional)..." (copied below). Suggest to remove the one in "2) (optional).."

"Since the OCI referrers API was defined in the not-yet-released [OCI Distribution Spec v1.1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md), very few registries support it."

Reply:
I intentionally duplicated some of the wording so that the requirements section is like the summary version and the comparison is the detailed version. I can agree that it's probably not adding enough value, though. I will remove it.